### PR TITLE
sdi-1070 handle create hearing for adjudication sync

### PR DESCRIPTION
**Pull request by lbennett-moj**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjudications/AdjudicationsApiService.kt [hunk 1]
```diff
@@ -3,12 +3,12 @@ package uk.gov.justice.digital.hmpps.prisonertonomisupdate.adjudications
 import org.springframework.stereotype.Service
 import org.springframework.web.reactive.function.client.WebClient
 import org.springframework.web.reactive.function.client.awaitBody
-import uk.gov.justice.digital.hmpps.prisonertonomisupdate.adjudications.model.ReportedAdjudicationResponseV2
+import uk.gov.justice.digital.hmpps.prisonertonomisupdate.adjudications.model.ReportedAdjudicationResponse
```
Comment (lbennett-moj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjudications/AdjudicationsApiService.kt:6]: rename in adjudications service

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjudications/AdjudicationsMappingService.kt [hunk 2]
```diff
@@ -40,11 +40,16 @@ class AdjudicationsMappingService(
...
+
+class AdjudicationMappingDto(
   val adjudicationNumber: Long,
   val chargeSequence: Int,
   val chargeNumber: String,
   val label: String? = null,
   val mappingType: String = "ADJUDICATION_CREATED",
   val whenCreated: LocalDateTime? = null,
-)
+  type: String = "ADJUDICATION",
```
Comment (lbennett-moj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjudications/AdjudicationsMappingService.kt:54]: will make this an enum in a future PR

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjudications/HearingsMappingService.kt [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      .awaitBody()
+}
+
+class AdjudicationHearingMappingDto(
+  val nomisHearingId: Long,
+  val dpsHearingId: String,
+  val label: String? = null,
+  val mappingType: String = "ADJUDICATION_CREATED",
+  val whenCreated: LocalDateTime? = null,
+  type: String = "HEARING",
```
Comment (lbennett-moj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjudications/HearingsMappingService.kt:49]: will rename when I've thought of something better

File: src/main/resources/application-localstack.yml [hunk 4]
```diff
@@ -35,7 +35,7 @@ hmpps.sqs:
       queueName: hmpps_prisoner_to_nomis_adjudication_queue
       dlqName: hmpps_prisoner_to_nomis_adjudication_dlq
       subscribeTopicId: domainevents
-      subscribeFilter: "{\"eventType\":[\"adjudication.report.created\"]}"
+      subscribeFilter: "{\"eventType\":[\"adjudication.report.created\", \"adjudication.hearing.created\"]}"
```
Comment (andymarke) [inline src/main/resources/application-localstack.yml:38]: I think damages must be missing here as well 🤔

**Review by andymarke — APPROVED** [review]
LGTM

**Pull request merged**